composer.json

```diff
@@ -116,6 +113,9 @@
         "[web-root]/web.config": false
       }
     },
+    "drupal-core-vendor-hardening": {
+      "drupal/module_name": ["tests", "src/Tests"]
```

Comment:
The module name should be real one

Reply:
At least core tests & umami profile could be removed

Reply:
For this to work we need to create a patch for 9.0 in https://www.drupal.org/project/drupal/issues/3108262

Reply:
Actually we can't apply patches in d/o issue cause composer uses a mirror github repo at https://github.com/drupal/core-vendor-hardening which only includes a subfolder of Drupal's gitlab repo so I will be removing this extra config.
We can add removal for core tests and umami once d/o issue gets fixed.